Caching.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26730.10
+# Visual Studio Version 16
```

Comment:
Let's keep this file compatible with VS 2017 until VS 2019 has a public preview.